static/js/utils/getInitAnswersFrame.js

```diff
@@ -0,0 +1,29 @@
+import { generateIFrame } from '../iframe-common';
+
+/**
+ * Returns a function which generates an iframed Answers experience.
+ * If the function is not called by the timeout, a warning is logged.
+ * 
+ * @param {string} domain 
+ * @param {HitchhikerConfigManager} hitchhikerConfigManager 
+ * @returns {Function}
+ */
+export default function getInitAnswersFrame (domain, hitchhikerConfigManager) {
```

Comment:
would something like generateInitAnswersFrame be better? We're not getting a preexisting function but generating a new one

Reply:
ya, that works